recipes/jill/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "jill" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jill-{{ version }}.tar.gz
+    sha256: 48ea86a508f87dab0293d00634b4e8c6677cb00676e617b6c46345bc10af5e10
+  #- url: https://raw.githubusercontent.com/johnnychen94/jill.py/master/requirements.txt
+
+build:
+  number: 0
+  noarch: python
+  script: 
+    ## Ensure inclusion of requirements.txt
+    {% set reqfile = "requirements.txt" %}
+    {% set reqsource = name ~ ".egg-info/requires.txt" %}
+    # IF {{ reqfile }} is NOT PRESENT, COPY {{ reqfile }} from {{ reqsource }}
+    - echo -e "\n>>> ENSURE INCLUSION OF '{{ reqfile }}' <<<\n\n"
+    - if [ ! -f {{ reqfile }} ] && [ -f {{ reqsource }} ]; then cp {{ reqsource }} {{ reqfile }}; fi  # [not win]
+    #- copy {{ name }}.egg-info\requires.txt {{ reqfile }}  # [win]
+    - copy {{ reqsource | replace('/', '\\') }} {{ reqfile }}  # [win]
+    ## Install package
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - wget
```

Comment:
Is this needed in host? If pip check fails it's better to either patch it so `wget` is not checked or remove pip check.

Reply:
Have not had created a patch yet. Have seen some documentation on patching though. But I don't have clarity on what goes on technically when a patch is used. Could you please elaborate on that? (perhaps with an example will be very helpful).

Thank you.

Reply:
Host are packages available during build and is primarily important when building against something or when cross-compiling so not this situation.

Pip check just checks if all packages can be discovered by pip. This requires, IIRC, that the package was built with pip in the first place which is not always the case. So pip check can fail even if the package is installed correctly. This also explains why you can import it but the check still fails.

A patch is just a normal git patch that is applied to your source: https://git-scm.com/docs/git-format-patch
But the easiest approach is probably to just remove pip check here. it's not mandatory to have it here.